src/main/java/io/jenkins/plugins/gitlabbranchsource/GitLabSCMSource.java

```diff
@@ -542,7 +542,7 @@ protected SCMRevision retrieve(@NonNull String thingName, @NonNull TaskListener
                 GitLabApi gitLabApi = apiBuilder(serverName);
                 listener.getLogger().format("Looking up project %s%n", projectPath);
                 gitlabProject = gitLabApi.getProjectApi().getProject(projectPath);
-                result.add(new ObjectMetadataAction(null, gitlabProject.getDescription(),
+                result.add(new ObjectMetadataAction(gitlabProject.getNameWithNamespace(), gitlabProject.getDescription(),
```

Comment:
The display name is updated but the description of gitlab project doesn't seem to be updated. :/

Reply:
Maybe I miss understand the object metadata for owner and repo 😓

Reply:
I'll have to investigate 😓

Reply:
Does not work at all 😭